### ISSUE TITLE
"error: cannot decrypt" on iOS

### ISSUE THREAD

**Issue by kryten87** [bitwarden/android#3179]
### Steps To Reproduce

Bitwarden App Version: 2024.4.0 (7272)
IOS version 17.4.1 iPhone SE

All items in vault show as "[error: cannot decrypt]"

I have
1. Locked and unlocked
2. Logged out and logged back in
3. Manually synced
4. Uninstalled and reinstalled app (and deleted data)
With no improvement

Data works in Chrome extension on desktop and via web interface on desktop and in iOS Chrome

### Expected Result

Should be able to search, view, copy passwords

### Actual Result

Unable to use app

### Operating System

iOS

### Operating System Version

17.4.1

### Device

IPhone SE (2nd generation)

### Build Version

Version: 2024.4.0 (7272)

### Beta

- [ ] Using a pre-release version of the application.

**Comment by xafarr** [comment]
Thanks @kryten87 raising this issue. I'm having the same problem.

@SergeantConfused For me it's happening for items in organisation.

**Comment by Hallix-SH** [comment]
Hi @kryten87, thanks for bringing this up. I'm having exactly the same problem. Bitwarden works as it should on all the other platforms I'm using: Android and Windows 11. But on IOS 17.4.1 I can only read the names of folders and types. Every other entry like the name of favourites, collections and every single password is [error: cannot decrypt].

I have tried the following:
- uninstalled - reinstalled the app on IOS
- changed my master password
- changed the encryption settings

But the problem hasn't changed in any way.

I also tried Bitwarden support, but the answer I got was useless. 

So any help or advice would be greatly appreciated.

**Comment by c0ffee** [comment]
I have the same issue, anyone has a workaround for this?

**Comment by unmerrymelodies** [comment]
Same issue here. On iOS all my items, including collections, within my organization are coming up with "error: cannot decrypt" All the same troubleshooting steps have been taken with no effect.

**Comment by Hallix-SH** [comment]
My impression is that the user is somehow defective. Now I have created a new user, a copy of the ‘defective’ one, so to speak, and everything is fine again. The question remains: What has broken?
As I'm sure nobody can answer the question of whether this also happens with other users in my teams, with individual users or all of them, I have set up a second password manager. I decided to use Proton Free as a backup. Exporting and importing worked and now I have to update Proton Free from time to time.

**Comment by c0ffee** [comment]
Could it be that this only happens to owner of an organization?

I have several accounts and other work fine only my main user who is the owner of the organization is affected on the iPad and iOS app

In web and on the Mac all works fine

This is really bad especially because it is open for so long now

**Comment by Hallix-SH** [comment]
That is very good point. It happend also in my case with me, the owner of the organization.

**Comment by fredrichallstrom** [comment]
I have a normal user that has the error.

**Comment by n8henrie** [comment]
I have the same issue on my wife's iPhone. For her, all shared items in our organization show up as `[error: cannot decrypt]`. Those same items show up fine for me, and the non-shared items on her account also show up fine.

I've also tried the same trouble shooting steps as OP (and even deleted and reinstalled the app completely, no change).

After a few weeks, I opened a support ticket on 5/3/24 and was told:

> This bug has been reported to our Engineering team for review - at this time I don't have an ETA for a fix, but I would expect to see one introduced in an upcoming version release to Bitwarden.

Unfortunately the CS rep had no recommendations for workarounds other than "use the web vault instead."

**Comment by unmerrymelodies** [comment]
I think Hallix-SH is correct that it is related to the account itself. I ran into this error because I recently created an organization to share items with my wife and found that on my phone I couldn’t access any of the items I shared. She was able to access everything fine. As someone thought it could be related to ownership status, I had her remove me as an owner for our org. Still didn’t work. I had her create her own org and invite me. Same issue with the org she created. I did an export of my account, deleted it, recreated it, imported my previous items and created a new org. Everything worked as it should. My wife re-added me to our org and it also worked as it should. 

Long story short, it seems the issue was with my account. It was kind of a pain recreating my account, but really it was only about 5 minutes of work and I was back in business.

**Comment by vvolkgang** [comment]
Issue migrated to https://github.com/bitwarden/mobile/issues/3179
